samples/job-shop-scheduling/README.md

```diff
@@ -0,0 +1,67 @@
+---
+page_type: sample
+languages:
+- python
+author: anraman
+ms.author: anraman@microsoft.com
+ms.date: 01/25/2021
+products:
+- azure-quantum
+- azure-qio
+description: "Solve a job shop scheduling problem using the Azure Quantum optimization service"
+urlFragment: azure-quantum.job-shop-problem
+---
+
+# Job Shop Scheduling Sample
```

Comment:
The way samples are rendered on MS Docs, it will help SEO to add Azure Quantum -- or some brand/tech -- to the H1.

Reply:
Ah, got it; I was using the `description` metadata property to generate the SEO keywords on docs.microsoft.com/samples, but I'll make sure that they appear in the H1s as well.